stdlib/2/multiprocessing/pool.pyi

```diff
@@ -17,7 +14,7 @@ class IMapIterator(Iterator[Any]):
 
 class IMapUnorderedIterator(IMapIterator): ...
 
-class Pool(ContextManager[Pool]):
+class Pool:
```

Comment:
This is a new-style class:

```suggestion
class Pool(object):
```

Reply:
Thanks, I guess I'm spoiled by Python 3 :smile:

Reply:
I know the feeling.